### ISSUE TITLE
bug: When tianmu_insert_delayed=0, 128 is inserted to tinyint unsigned type, but the value range is [0,127]

### ISSUE THREAD

**Issue by adofsauron** [stoneatom/stonedb#1236]
### Have you read the Contributing Guidelines on issues?

- [X] I have read the [Contributing Guidelines on issues](https://github.com/stoneatom/stonedb/blob/stonedb-5.7-dev/CONTRIBUTING.md).

### Please confirm if bug report does NOT exists already ?

- [X] I confirm there is no existing issue for this

### When tianmu_insert_delayed=0, 128 is inserted to tinyint unsigned type, but the value range is [0,127]

```sql

create table tiny(a tinyint, b tinyint unsigned) engine = tianmu DEFAULT CHARSET=utf8mb4;

```

### When tianmu_insert_delayed=0

```sql

mysql> insert into tiny values(0, 128);
Query OK, 1 row affected (0.01 sec)


```

```sql


mysql> desc tiny;
+-------+---------------------+------+-----+---------+-------+
| Field | Type                | Null | Key | Default | Extra |
+-------+---------------------+------+-----+---------+-------+
| a     | tinyint(4)          | YES  |     | NULL    |       |
| b     | tinyint(3) unsigned | YES  |     | NULL    |       |
+-------+---------------------+------+-----+---------+-------+
2 rows in set (0.00 sec)


```

```sql

mysql> select * from tiny;
+------+------+
| a    | b    |
+------+------+
| -128 |    0 |
|  127 |  127 |
|    0 |  127 |
|    0 |    0 |
|    0 |    0 |
|    0 |  128 |
|    0 |  128 |
|    0 |  128 |
|    0 |  128 |
|    0 |  128 |
|    0 |  128 |
+------+------+
11 rows in set (0.00 sec)


```

### When tianmu_insert_delayed=1

```sql

mysql> insert into tiny values(0, 128);
ERROR 1264 (22003): Out of range[0, 127] for column 'b' value: 128

```

### Are you interested in submitting a PR to solve the problem?

- [ ] Yes, I will!

**Comment by isredstar** [comment]
Be similar to #1082

**Comment by hustjieke** [comment]
related to issue #1218 and fixed in pr #1220
